release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,16 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 12 June 2023 - Fix - CloudGateway 2.11.0 - Resolved concurrency issue [ID_36439]
+
+Cloud connection will remain active when the DMA name no longer corresponds to the server name after rename.
+Cloudgateway uses now the DMA name as defined in dataminer.xml or from DMZ options instead of the server name to verify its state.
```

Comment:
```suggestion
DataMiner CloudGateway now uses the DMA name as defined in dataminer.xml or from the DMZ options instead of the server name.
```

Reply:
Add what issue/bug this fixes: remote access, sharing etc showing dms unavailable error on the blue page